### ISSUE TITLE
Invalid Count Value Error.

### ISSUE THREAD

**Issue by jpwijbenga** [DotJoshJohnson/vscode-xml#291]
#### Description
Formatter gives cryptic error message: "Invalid Count Value" when trying to format small XML. Not sure if the document is faulty or formatter has a bug.
```
<?xml version="1.0" encoding="UTF-8"?>
<a>
	The Total (<value-of />) is not equal to the sum (<value-of />).
</a>
```

#### Formatter Implementation
v2

#### XML Tools Version
2.5.0

#### VS Code Version
1.40.1 User

#### Operating System
Win10 Ent (x64) 1809

**Comment by SebastianZ** [comment]
This seems to be a bug in the formatter.

Here's another simple test case:
```xml
<x>foo<y><z>bar</z></y></x>
```

Trying to format this also results in an "Invalid count value" error.

Though if you insert a line break after "foo" like this

```xml
<x>foo
<y><z>bar</z></y></x>
```

the XML will be formatted correctly.

Sebastian

**Comment by DotJoshJohnson** [comment]
This should be fixed by #317.

**Comment by SebastianZ** [comment]
I can confirm that this issue is gone in the latest version.

The output of my example looks like this now after formatting:

```xml
<x>foo<y>
  <z>bar</z>
</y>
</x>
```

I had expected it to format it like this:

```xml
<x>foo
  <y>
    <z>bar</z>
  </y>
</x>```

Though at least the error is gone, so thank you for fixing this!

Sebastian
